config/repository_resource.go

```diff
@@ -311,6 +311,9 @@ func (r *repositoryResource) enforceState(ctx context.Context) (inDesiredState b
 	return true, nil
 }
 
+func (r *repositoryResource) populateOutput(rCompliance *agentendpointpb.OSPolicyResourceCompliance) {
+}
```

Comment:
Why are these empty `populateOutput()` functions needed and why is this one formatted differently?

Reply:
No idea why it was formatted this way, ill see if I can fix that manually. The reason for the empty methods is so that each resource type implements the interface.

Reply:
Welp the formatter wants it this way, can't really argue with gofmt